bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/ITCRequest.scala

```diff
@@ -185,8 +187,11 @@ object ITCRequest {
   def gsaoiParameters(r: ITCRequest): GsaoiParameters = {
     val filter      = r.enumParameter(classOf[Gsaoi.Filter])
     val readMode    = r.enumParameter(classOf[Gsaoi.ReadMode])
-    val gems        = gemsParameters(r)
-    GsaoiParameters(filter, readMode, gems)
+    val iq          = obsConditionParameters(r).iq
+    ConfigExtractor.createGsaoiParameters(filter, readMode, iq) match {
+      case \/-(p) => p
+      case -\/(t) => throw new IllegalArgumentException(t)
+    }
   }
```

Comment:
You can also delete `gemsParameters` from this class as it isn't being used anymore.

I should have pointed that out, but I guess I completely forgot.

Reply:
It is deleted in my local repository, tried to push newest changes, but git tells me there's nothing new to commit.

Reply:
Did you do a `commit -am "REL-3222: Removed gemsParameters"` first?

I'm looking at the file in this commit and I still see:
```
def gemsParameters(r: ITCRequest): GemsParameters = {
    val avgStrehl  = r.doubleParameter("avgStrehl") / 100.0
    val strehlBand = r.parameter("strehlBand")
    GemsParameters(avgStrehl, strehlBand)
}
```

in `ITCRequest.scala`.

Reply:
Just re-committed changes now.